guide/simulation-export.md

```diff
@@ -0,0 +1,10 @@
+# Simulation Export
+
+This chapter describes how to export Webots simulations to external applications.
```

Comment:
Instead of "to external applications", I would say "to X3D format and web pages".